packages/toc/src/toc.tsx

```diff
@@ -24,6 +26,26 @@ import { TOCTree } from './toc_tree';
  */
 const RENDER_TIMEOUT = 1000;
 
+/**
+ * Adds commands to command registry.
+ *
+ * @param commands - command registry
+ * @param toc - List of headings
+ * @returns
+ */
+const addCommands = (commands: CommandRegistry, toc: IHeading[]) => {
+  commands.addCommand(CommandIDs.runCells, {
+    execute: args => {
+      const pos = args['pos'] as number;
+      const heading = toc[pos];
+      let code: INotebookHeading[];
+      code = [];
+      return NestedCodeCells(toc, heading, code);
+    },
+    label: 'Run Cell(s)'
```

Comment:
Here @goanpeca.

Reply:
I am seeing that some parts of codebase use trans here but others don't. I am thinking that maybe it is not needed if trans could be called in `addCommand`? I don't know.

![Screenshot from 2021-05-20 16-23-42](https://user-images.githubusercontent.com/5832902/119005928-cf9f9e00-b987-11eb-9b59-d74446a56c4b.png)